<launch>
  <node pkg="vehicle_detection" name="vehicle_tracker" type="vehicle_tracker" output="screen"/>
  <rosparam file="$(find vehicle_detection)/vehicle_detection.yaml" />
</launch>
